contracts/colony/ColonyFunding.sol

```diff
@@ -205,6 +205,27 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
     return (fundingPot.associatedType, fundingPot.associatedTypeId, fundingPot.payoutsWeCannotMake);
   }
 
+ function moveFundsBetweenPots(
+    uint256 _permissionDomainId,
+    uint256 _childSkillIndex,
+    uint256 _domainId,
+    uint256 _fromChildSkillIndex,
+    uint256 _toChildSkillIndex,
+    uint256 _fromPot,
+    uint256 _toPot,
+    uint256 _amount,
+    address _token
+  )
+  public
+  stoppable
+  authDomain(_permissionDomainId, _childSkillIndex, _domainId)
+  authDomain(_domainId, _fromChildSkillIndex, getDomainFromFundingPot(_fromPot))
```

Comment:
Looking at this again, I'm realizing that I don't see how this works -- if the caller has the permission in `_permissionDomainId`, then how can this call succeed? Even if the index is correct, the caller may not have the permission in `_domainId` and so this check fails.

Reply:
You're absolutely right, the second two `authDomain`s should only be `validateDomainInheritance`, and not the full `authDomain`s